docker/Dockerfile_ubuntu_deps

```diff
@@ -43,17 +43,37 @@ RUN . ./etc/os-release && \
 		cmake \
 		gfortran
 
+
+
 ENV AV_DEV=/opt/AliceVision_git \
     AV_BUILD=/tmp/AliceVision_build \
     AV_INSTALL=/opt/AliceVision_install \
     PATH="${PATH}:${AV_BUNDLE}"
 
+
+
+
+
 COPY dl/vlfeat_K80L3.SIFT.tree ${AV_INSTALL}/share/aliceVision/
 RUN echo "export ALICEVISION_VOCTREE=${AV_INSTALL}/share/aliceVision/vlfeat_K80L3.SIFT.tree" > /etc/profile.d/alicevision.sh
 
 COPY CMakeLists.txt ${AV_DEV}/
 
-COPY dl/deps ${AV_BUILD}/external/download/
+WORKDIR ${AV_DEV}
+
+COPY docker ${AV_DEV}/docker/
+
+RUN ls ${AV_DEV}/docker
+
+RUN pwd
+
+RUN ./docker/fetch.sh
```

Comment:
I'm not sure it's a good idea to move the download of the dependencies in the dockerfile or not. What are the advantages to do that?

In all case, it's a bit unrelated to the original PR goal and should at least be done in a separated commit as it's a different feature. I cannot merge the cpu checks for now due to that.

Reply:
If we put the download in the docker, we cannot cache the download itself, it will be invalidated if we change something in the dockerfile.

Reply:
I understand the problem with moving the download into docker. 

The reason I put fetch.sh into the dockerfile initially was that the user machine would end up requiring cmake in order to run fetch.sh. But I have another solution that I can implement in a different PR. 

I can make the fetch.sh run inside docker with the same uid as the running user. And then mount the current directory as a volume inside of the cmake container (let's say). This will allow us to cache the file, and the user will only need docker to run the build, no other dependencies.

Reply:
> If we put the download in the docker, we cannot cache the download itself, it will be invalidated if we change something in the dockerfile.

I will remove the changes now, and only put the cpu-check.sh changes.

Reply:
Thanks !
Btw, could you do the same modification for centos dockerfile?

Reply:
Done

Reply:
Thanks!!